packages/math/init.lua

```diff
@@ -16,6 +16,8 @@ function package:_init ()
       return value * SILE.settings:get("math.font.size") / 18
     end
   })
+  self:loadPackage("counters")
+  SILE.scratch.counters.equation = { value = 0, display = "arabic" }
```

Comment:
Maybe we should avoid tapping into "scratch" variables (I know we do it in a lot of places, but it's kind of annoying IMHO to expose such internals at package boundaries).
Wouldn't `SILE.call("set-counter", { id="equation" })` just work?

Reply:
(Or even do nothing, increment-counter will create the counter if needed)

Reply:
Thank you for your insights. In fact, in my first attempt, I did not initialize the counter (didn't even load the counters package), and everything worked fine. But I thought that having equation set to 0 would be better, for whatever reason.